static/js/hashchange.js

```diff
@@ -133,14 +148,18 @@ function do_hashchange_normal(from_reload) {
         case "#streams":
         case "#organization":
         case "#settings":
-        case "#recent_topics":
```

Comment:
There's a broken intermediate state in here, where various other items have the `recent_topics` code running.

Reply:
Fixed in the commit where it's introduced and resolved the merge conflict when you fix it ~4 commits later :)